Lib/test/test_email/test_message.py

```diff
@@ -486,11 +486,15 @@ def message_as_iter_attachment(self, body_parts, attachments, parts, msg):
         attachments = [allparts[n] for n in attachments]
         self.assertEqual(list(m.iter_attachments()), attachments)
 
+    def _is_multipart_msg(self, msg):
+        return 'Content-Type: multipart' in msg
+
     def message_as_iter_parts(self, body_parts, attachments, parts, msg):
         m = self._str_msg(msg)
         allparts = list(m.walk())
         parts = [allparts[n] for n in parts]
-        self.assertEqual(list(m.iter_parts()), parts)
+        iter_parts = list(m.iter_parts()) if self._is_multipart_msg(msg) else []
```

Comment:
This line is just over 80 chars long.

Reply:
Vim tells me it's exactly 80 chars :-)

Reply:
With the other change, the line is now shorter - 75chars. 

By the way there's 17 other lines longer than 79chars in this file, this leads me to think the limit is somewhat relaxed in the test suite?

Reply:
Thanks. Things slip through, we try to get it right in new changes.